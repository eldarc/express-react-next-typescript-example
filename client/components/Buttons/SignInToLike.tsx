import * as React from 'react'
import { Button } from 'reactstrap'
import Link from 'next/link'

const SignInToLike: React.FunctionComponent = () => {
  return (
    <Link href="/signin">
      <Button className="signin-to-like-btn" size="sm" color="primary">
        Sign in to like
      </Button>
    </Link>
  )
}

export default SignInToLike
