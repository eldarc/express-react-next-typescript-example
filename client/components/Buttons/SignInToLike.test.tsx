/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import SignInToLike from './SignInToLike'

describe('SignInToLike', () => {
  it('renders correctly', () => {
    const tree = renderer
        .create(<SignInToLike/>)
        .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
