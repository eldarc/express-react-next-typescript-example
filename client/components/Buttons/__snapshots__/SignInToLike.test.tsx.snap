// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInToLike renders correctly 1`] = `
<button
  aria-label={null}
  className="signin-to-like-btn btn btn-primary btn-sm"
  onClick={[Function]}
  type="button"
>
  Sign in to like
</button>
`;
